src/Symfony/Component/Lock/Store/StoreFactory.php

```diff
@@ -34,6 +34,9 @@ public static function createStore($connection)
         if ($connection instanceof \Memcached) {
             return new MemcachedStore($connection);
         }
+        if ($connection instanceof \Zookeeper) {
+            return new ZookeeperStore($connection);
```

Comment:
This is not listed as return type and parameter type (yet). It's a bit of an odd construction though, way too many types possible for both the parameter and return.

Reply:
Added the parameter and return type.